source/runbooks/dr-process.html.md.erb

```diff
@@ -0,0 +1,355 @@
+
+
+## Platform rebuild - Networking
+
+
+This is a large undertaking, and has not been tested but discussed within the team.
+
+
+### Networking Overview
+
+
+Our [Networking Diagram](../concepts/networking/networking-diagram.html) shows a high level view of how shared
+VPCs are connected.
+
+
+Our [NetworkingApproach](../concepts/networking/networking-approach.html) discusses and
+contains a detailed view of our shared VPCs.
+
+
+Our [Transit Gateway](../how-vpcs-access-the-internet.html) and how VPCs access the internet.
+
+
+
+## Stages
+
+Rather than rebuilding the platform through workflows, I would run steps manually to easier debug issues, however, technically, you could start to run the account creation process with our github workflows.
+
+Located [here](https://github.com/ministryofjustice/modernisation-platform/actions), on the left you will see the deployment workflows for all core based accounts, required for the building on the platform.
+
+
+### VPCs and Subnets
+
+
+1. Clone the mod-platform repo, change region in code. Whilst its as simple as a find and replace, modules are called and need changing, either locally and referenced locally, or in the module and pushed to git.
+2. Core VPC rebuild. Provider would need to be eu-west-1, so check the permissions you have in that region with the credentials you are using. E.g. https://moj.awsapps.com/start/ when getting your credentials from here, make sure the region is correct.
+3. Core Networking would be the next to run terraform on, checking the code and modules it uses for region references and changing them.
+
+
+**With the infrastructure in London no longer available, we would use the original ranges**
+
+
+### Transit Gateway
+
+
+To rebuild the transit gateway in a new region, first find the code, located here, in core-network-services. https://github.com/ministryofjustice/modernisation-platform/tree/main/terraform/environments/core-network-services
+
+
+Using the admin role, run a terraform plan and apply. This hasnt been done from scratch since the platforms creation, so expect to debug issues.
+
+
+### Firewall - Needs work
+
+
+We make use of AWS Network Firewalls in two ways. First, we use a centralised inspection VPC to control traffic entering and exiting the Modernisation Platform to and from internal Ministry of Justice services. Second, we use inline-inspection in our egress VPCs to control traffic exiting the Modernisation Platform to the internet.
+
+
+Located in the Core-Network-Services account, DNS, networking and other network related infrastructure will be down.
+
+
+The firewall rules can be located here - https://github.com/ministryofjustice/modernisation-platform/tree/main/terraform/environments/core-network-services/firewall-rules
+
+
+Live data & non-live data.
+
+
+Peer the new Transit grateway build in eu-west-1 to the MOJ transitgateway that is in eu-west-2 ???
```

Comment:
```suggestion
Peer the new Transit gateway build in eu-west-1 to the MOJ transit gateway that is in eu-west-2 ???
```

Reply:
Yes, not sure on that one, if our TGW is down due to a regional disaster, so is MOJ's TGW. This will require more coordinating with the network ops team and with the service owners. There will need to be other workarounds done for this.
Worth capturing it that there are dependencies on other teams and that peering with another TGW will be most likely impossible here.